src/Assert.php

```diff
@@ -1716,10 +1716,7 @@ public static function isMap($array, $message = '')
     {
         if (
             !\is_array($array) ||
-            !$array ||
-            \array_keys($array) !== \array_filter(\array_keys($array), function ($key) {
-                return \is_string($key);
-            })
+            \array_keys($array) !== \array_filter(\array_keys($array), 'is_string')
```

Comment:
Would using `'\is_string'` make it always opchachabel etc, or doesn't that work in this context?

Reply:
Strings are always to the FQN form 👌